### DISCUSSION TITLE
How to transcribe long audio to srt file directly?

### DISCUSSION THREAD

**Discussion by jasonsu123** [openai/whisper#1690] [Q&A]
Hello everyone, 
I have a question. I'd like to use the Whisper API to transcribe approximately 2 hours of a conference speech from an mp4 video into an srt subtitle file. I have a few questions:
Do I need to convert the mp4 file into wav or mp3 format first?

It seems that the Whisper API has a file size limit of 25 MB per processing. 
If I split the video into chunks, the resulting srt file might have incorrect timecodes. 

How should I handle this? 

Thank you.

**Comment by phineas-pta** [comment]
your question should be asked at: https://community.openai.com/tag/whisper

**Reply by jasonsu123** [reply]
Thank you for your suggestion. Just to clarify, can I post questions related to the use of openAI APIs on this website after registering and logging in?

**Reply by phineas-pta** [reply]
yes it's for posting that kind of questions

the github here is to run locally
